Pipfile

```diff
@@ -48,7 +48,7 @@ requests-mock = "~=1.9.3"
 tblib = "~=1.7.0"
 flake8 = "*"
 mypy = "*"
-black = "*"
+black = "~=22.12.0"
```

Comment:
I think I'd rather have black unpinned and allow it to reformat some files now.

Plus I don't think you should be seeing reformatting applied to `workshops/views.py` since this file is not touched by your changes and would not trigger pre-commit hook.

Reply:
I wrote the wrong name 🤦 it was reformatting `workshops/models.py`. As I've reverted it and not made further changes to that file, the formatting won't change now anyway.

Perhaps we should bump our CI linting, pre-commit file etc. to use black 23.x and reformat the whole repo accordingly once we have v4.1 ready to go?

Reply:
@elichad 
> Perhaps we should bump our CI linting, pre-commit file etc. to use black 23.x and reformat the whole repo accordingly once we have v4.1 ready to go?

That's a good idea.